src/styles/css/styles.css

```diff
@@ -810,7 +810,7 @@ td.ant-table-column-sort {
 .ant-card-body {
   background-color: #262626;
   border-radius: 50px 10px;
-  width: 400px;
+  width: fit-content;
```

Comment:
What did `400px` cause the code to do and how does `fit-content` fix that issue?

Reply:
Sorry I should have explained. I have run this all by Ash and he said it is good. The card-body that is changed was the card the entire Case Details page was being rendered on. It was jamming it all into the small 400px width. By setting it to fit-content it is allowing it all to display properly full page and not all jammed to one side.